src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -159,7 +159,7 @@ def register_common_storage_account_options(self):
                       min_api='2017-06-01',
                       help='Generate and assign a new Storage Account Identity for this storage account for use '
                            'with key management services like Azure KeyVault.')
-        self.argument('access_tier', arg_type=get_enum_type(t_access_tier),
+        self.argument('access_tier', arg_type=get_enum_type([tier for tier in t_access_tier if tier!= 'Cold']),
```

Comment:
Why disable `Cold` tier from client side?

Reply:
SDK added this enum which is still only available in some regions, will reenable cold tier when it is GA in all public regions.